### ISSUE TITLE
Checkpoints for COCO and VOC

### ISSUE THREAD

**Issue by alexlioralexli** [facebookresearch/simsiam#19]
Is it possible to share trained checkpoints for the SimSiam models finetuned on COCO and VOC?

**Comment by endernewton** [comment]
I think you can directly do the fine-tuning on COCO/VOC using the released code? We do not have plans to release the trained Mask R-CNN models, but it should be easy to reproduce.
